src/query/src/error.rs

```diff
@@ -126,7 +126,7 @@ pub enum Error {
         location: Location,
     },
 
-    #[snafu(display("DataFusion error"))]
+    #[snafu(display(""))]
```

Comment:
It'd be better not to provide an empty display message. If some places use `to_string()` or `Display` to show the error, we will get an empty message.

Reply:
It seems that new DataFusion can pushdown functions on "limit" clause, then the error messages could be different between standalone and distributed modes, causing sqlness to fall. See this action run before I changed this error: https://github.com/GreptimeTeam/greptimedb/actions/runs/14212982871/job/39823883300

To pass sqlness, I have to either add "DataFusion error" in `PollStreamSnafu`'s display, and produces a lot of verbose "DataFusion error:" in the error output; or do it like this here, and some verbose “DataFusion error:" are reduced as well. I choose the latter for better message conciseness.

Reply:
> It seems that new DataFusion can pushdown functions on "limit" clause, then the error messages could be different between standalone and distributed modes, causing sqlness to fall.

You can move inconsistent test results to standalone and distributed tests separately.

Reply:
that considered and less desired because i thought it does not worth the cost of duplicating the codes, and what's more important, now the error messages are less verbose

Reply:
i may find a way to make `to_string` on these errors not accidently produce empty result while still keeping the sqlness happy, will do it in next pr